File: php/prime.php
<!DOCTYPE html>
<html>
<head>
    <title>Prime Number Checker</title>
</head>
<body>
    <form method="post">
        Enter a number: <input type="number" name="num">
        <input type="submit" value="Check">
    </form>

    <?php
    if ($_SERVER["REQUEST_METHOD"] == "POST") {
        $num = $_POST['num'];
        $isPrime = true;

        if ($num <= 1) {
            $isPrime = false;
        } else {
            for ($i = 2; $i <= sqrt($num); $i++) {
                if ($num % $i == 0) {
                    $isPrime = false;
                    break;
                }
            }
        }

        if ($isPrime) {
            echo "$num is a Prime Number";
        } else {
            echo "$num is Not a Prime Number";
        }
    }
    ?>
</body>
</html>

File: java/Prime.java
import java.util.Scanner;
public class Prime {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a positive integer: ");
        int number = scanner.nextInt();
        boolean flag =true;
        for(int i=2;i<=number/2;i++){
            if(number%i==0){
                flag=false;
            }else{
                flag=true;
            }
        }
        if(flag==true){
            System.out.println(number + " is a prime number");
        }else{
              System.out.println(number + " is  not a prime number");
        }
    }
}
